### DISCUSSION TITLE
Micronaut Integration?

### DISCUSSION THREAD

**Discussion by aadrian** [pf4j/pf4j#539] [General]
Hi,

Is there any plan to have a Micronaut https://micronaut.io/ integration? 

Thank you.

**Comment by decebals** [comment]
No, I don't have a such plan.
First of all, it's not clear for me what this integration entails. Can you elaborate more on the subject, maybe someone has played with pf4j and Microunaut and can help?

**Reply by aadrian** [reply]
Micronaut can be seen as an alternative to Spring Boot (this also seem to not work https://github.com/pf4j/pf4j-spring/issues/77 )
An integration would allow to develop/produce "plugins" that could be included/disabled/etc, as opposed to hardcoding the functionality. 

Another framework (by the same team) https://grails.org/ has it's own plug-in concept (and there are many examples for "custom"/incopatible plugin systems).
So basically by having integrations, PF4J could act as a "standard" :) .

**Reply by decebals** [reply]
For SpringBoot and PF4J integration please take a look on https://github.com/hank-cp/sbp.

**Reply by aadrian** [reply]
> For SpringBoot and PF4J integration please take a look on https://github.com/hank-cp/sbp.

@decebals  Thank you for the link. Why not link it from the main page https://github.com/pf4j/pf4j  ?
